content/en/developers/ide_integrations/idea/_index.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Datadog Plugin For IntelliJ IDEA
+kind: documentation
+disable_toc: false
+is_beta: true
+further_reading:
+- link: "/account_management/api-app-keys/"
+  tag: "Documentation"
+  text: "Find out more about API and application keys."
+- link: "/getting_started/profiler/"
+  tag: "Documentation"
+  text: "Getting started with Continuous Profiler."
+---
+
+{{< beta-callout url="#" btn_hidden="true">}}
+  The Datadog plugin for IntelliJ IDEA is in Public Beta. It is intended for customers that use the <a href="https://docs.datadoghq.com/profiler/#pagetitle">Continuous Profiler</a> for their Java services. If the plugin stops working unexpectedly, check for plugin updates.
```

Comment:
maybe we should add, and reach out via email team handle?

Reply:
I've updated the banner language to make it a little less impersonal and linked to the feedback section below. That way, users can look at all the different feedback options in one place.